### PR TITLE
Get the PAT from the OS Keyring

### PR DESCRIPTION
**Pull request by gaborcsardi**

### CODE REVIEW

File: R/gh_token.R [hunk 1]
```diff
@@ -0,0 +1,165 @@
...
+#' You can read more about PATs here:
+#' <https://help.github.com/articles/creating-a-personal-access-token-for-the-command-line/>
+#' and you can access your PATs here (if logged in to GitHub):
+#' <https://github.com/settings/tokens>.
+#'
+#' Set the `GITHUB_PAT` environment variable to avoid having to include
+#' your PAT in the code. If you work with multiple GitHub deployments,
+#' e.g. via GitHub Enterprise, then read 'PATs for GitHub Enterprise' below.
+#'
+#' In you want a more secure solution than putting authentication tokens
```
Comment (jennybc) [inline R/gh_token.R]: "In" --> "If"

File: R/gh_token.R [hunk 2]
```diff
@@ -0,0 +1,165 @@
...
+#' into envirnment variables, read 'Storing PATs in the system keyring'
```
Comment (jennybc) [inline R/gh_token.R]: "environment"

**Comment by gh-testing** [comment]
Thanks!

**Comment by gaborcsardi** [comment]
Thanks!

**Pull request merged**